firefox-ios/Client/Utils/OhttpGleanUploader.swift

```diff
@@ -0,0 +1,92 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import Glean
+
+/// An enumeration representing different environments for the OHTTP client.
+public enum OhttpEnvironment {
+    case staging
+    case prod
+
+    // TODO: Laurie - config URL to pass into the OHTTPManager?
+    /// Returns the configuration URL based on the selected environment.
+    var config: URL? {
+        switch self {
+        case .staging:
+            return URL(string: "https://stage.ohttp-gateway.nonprod.webservices.mozgcp.net/ohttp-configs")
+        case .prod:
+            return URL(string: "https://prod.ohttp-gateway.prod.webservices.mozgcp.net/ohttp-configs")
+        }
+    }
+
+    // TODO: Laurie - relay URL to pass into the OHTTPManager?
+    /// Returns the relay URL based on the selected environment.
+    var relay: URL? {
+        switch self {
+        case .staging:
+            return URL(string: "https://mozilla-ohttp-dev.fastly-edge.com/")
+        case .prod:
+            return URL(string: "https://mozilla-ohttp-dev.fastly-edge.com/")
+        }
+    }
+
+    static func getEnvironment() -> OhttpEnvironment {
+        // TODO: Laurie - Double-check this is accurate
+        let shouldUseRelease = AppConstants.buildChannel == .release || AppConstants.buildChannel == .beta
```

Comment:
Shoud we use the release URLs for production and beta? Or should beta use the staging environment?

Reply:
For this work, lets forget about ohttp staging entirely. It was more useful for fakespot, but at this point we are happy with the production endpoints.